### PR TITLE
feat: add support for Terraform v0.15

### PR DESCRIPTION
**Pull request by soerenmartius**
- feat: upgrade pre-commit hooks to v0.2.3
- build: upgrade build-tools to v0.11.0
- feat: add support for terraform v0.15
- chore: prepare v0.5.0 release

### CODE REVIEW

**Review by mariux — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -607,7 +608,7 @@ Copyright &copy; 2020 [Mineiros GmbH][homepage]
 [badge-build]: https://github.com/mineiros-io/terraform-aws-s3-bucket/workflows/CI/CD%20Pipeline/badge.svg
 [badge-semver]: https://img.shields.io/github/v/tag/mineiros-io/terraform-aws-s3-bucket.svg?label=latest&sort=semver
 [badge-license]: https://img.shields.io/badge/license-Apache%202.0-brightgreen.svg
-[badge-terraform]: https://img.shields.io/badge/terraform-0.13%20and%200.12.20+-623CE4.svg?logo=terraform
+[badge-terraform]: https://img.shields.io/badge/terraform-0.15%20and%200.12.20+-623CE4.svg?logo=terraform
```
[the remark this answers is not in the document]
Comment (soerenmartius) [reply README.md:611]: nope, doesn't look appropriate to me - there  is absolutely no value in listing every single version. We should use a semver range instead. Also, a badge shouldn't      block the PR.

[the remark this answers is not in the document]
Comment (soerenmartius) [reply README.md:611]: nope, doesn't look appropriate to me - there  is absolutely no value in listing every single version. We should use a semver range instead. Also, a badge shouldn't      block the PR.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-[<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="400"/>][homepage]
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/
+j3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="400"/>][homepage]
```
Comment (mariux) [inline README.md:2]: thi sbreaks the image

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariux) [reply README.md:611]: also fine using semver but again 0.15 AND 0.12 is wrong. as it excludes 0.14 and 0.13

Comment (mariux) [inline README.md:611]:
```suggestion
[badge-terraform]: https://img.shields.io/badge/terraform-0.15%20|0.14%20|%200.13%20|%200.12.20+-623CE4.svg?logo=terraform
```

**Review by mariux — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (soerenmartius) [reply README.md]: resolved in 53f93dc

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (soerenmartius) [reply README.md]: resolved in 53f93dc

Comment (soerenmartius) [reply README.md]: resolved

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**